operator/app/src/test/java/org/keycloak/operator/ClusteringE2EIT.java

```diff
@@ -173,11 +177,14 @@ public void testKeycloakCacheIsConnected() throws Exception {
         // This is to test passing through the "Service", not 100% deterministic, but a smoke test that things are working as expected
         // Executed here to avoid paying the setup time again
         var service = new KeycloakService(k8sclient, kc);
-        Awaitility.await().atMost(5, MINUTES).ignoreExceptions().untilAsserted(() -> {
+        Awaitility.await()
+                .atMost(20, MINUTES)
+                .pollDelay(5, SECONDS)
+                .ignoreExceptions()
+                .untilAsserted(() -> {
             String token2 = null;
             // Obtaining the token from the first pod
-            // Connecting using port-forward and a fixed port to respect the instance issuer used hostname
-            for (int i = 0; i < (targetInstances * 2); i++) {
+            for (int i = 0; i < (targetInstances + 1); i++) {
```

Comment:
Just out of curiosity. Is it really necessary to decrease the number of test loops? It possibly makes the smoke test even more "smoky". 😅

Reply:
That's the intention, unfortunately, the cache takes quite some time to "connect" and stay reliable in all instances.
The alternative would be to turn those tests into even more "long-running" E2E.

Reply:
Ok, fair enough. 🙂 Even though I'm not particularly happy that we need to loosen the test, it's a reasonable trade-off.

Reply:
Deal 👍